frontend-ssr/components/tableOfContents/TableOfContents.module.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../styles/variables';
+@import '../../styles/buttons';
+.tableOfContents {
+    li {
+        list-style-type: none;
+        position: relative;
+        padding-left: 2.5em;
+        min-height: 5em;
+        button {
+            text-align: left;
+        }
+        &::before {
+            content: "";
+            position: absolute;
+            top: 0;
+            left: 0;
+            display: block;
+            width: 1em;
+            height: 1em;
+            border-radius: 100%;
+            border: 2px solid color(grey);
+            background-clip: content-box;
+            padding: 0.2em;
+        }
+        &::after {
+            content: '';
+            position: absolute;
+            top: 42%;
+            left: 0.8em;
+            background-color: color(grey);
+            width: 0.13em;
+            height: 50%;
+        }
+        &.active,
+        &:hover {
```

Comment:
There's a small usability problem if these get highlighted when hovering the `<li>`. This gives me the impression I can click them, however, I can only do it if I click on the `<button>`.

So, the expected behavior is that if the dot gets highlighted, I can also click it and scroll.

![2020-09-12_15h19_24](https://user-images.githubusercontent.com/11327761/92995335-5e137400-f50b-11ea-9a34-2ce7fd8f497d.gif)